config/cps-sink-connector.properties

```diff
@@ -1,6 +1,6 @@
 name=CPSSinkConnector
 connector.class=com.google.pubsub.kafka.sink.CloudPubSubSinkConnector
 tasks.max=10
-topics=my-kafka-topic
-cps.topic=my-cps-topic
-cps.project=my-cps-project
+#topics=my-kafka-topic
```

Comment:
Why are we commenting these out?

Reply:
I was planing to use something like `sed -i \"$ a topics=my-kafka-topic\n cps.topic=cloud\n cps.project=tz-playground-bigdata\n\" ~/config/cps-sink-connector.properties` to add a line in the config file from the startup script.

Reply:
Lets leave this as is for now, it shows users how to configure the properties file which is the primary purpose. If we need something different for the test, we can branch from this file, which I was planning to do in any case.

Reply:
I see similar things in the Kafka config files. I think it's pretty standard. See the last lines in `kafka_2.13-3.1.0/config/connect-standalone.properties`. 

We should describe in our example config properties what each options mean like in the Kafka example.

```
tianzi 11:12 kafka_2.13-3.1.0 % cat config/connect-standalone.properties 
# Licensed to the Apache Software Foundation (ASF) under one or more
# contributor license agreements.  See the NOTICE file distributed with
# this work for additional information regarding copyright ownership.
# The ASF licenses this file to You under the Apache License, Version 2.0
# (the "License"); you may not use this file except in compliance with
# the License.  You may obtain a copy of the License at
#
#    http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

# These are defaults. This file just demonstrates how to override some settings.
bootstrap.servers=localhost:9092

# The converters specify the format of data in Kafka and how to translate it into Connect data. Every Connect user will
# need to configure these based on the format they want their data in when loaded from or stored into Kafka
key.converter=org.apache.kafka.connect.json.JsonConverter
value.converter=org.apache.kafka.connect.json.JsonConverter
# Converter-specific settings can be passed in by prefixing the Converter's setting with the converter we want to apply
# it to
key.converter.schemas.enable=true
value.converter.schemas.enable=true

offset.storage.file.filename=/tmp/connect.offsets
# Flush much faster than normal, which is useful for testing/debugging
offset.flush.interval.ms=10000

# Set to a list of filesystem paths separated by commas (,) to enable class loading isolation for plugins
# (connectors, converters, transformations). The list should consist of top level directories that include 
# any combination of: 
# a) directories immediately containing jars with plugins and their dependencies
# b) uber-jars with plugins and their dependencies
# c) directories immediately containing the package directory structure of classes of plugins and their dependencies
# Note: symlinks will be followed to discover dependencies or plugins.
# Examples: 
# plugin.path=/usr/local/share/java,/usr/local/share/kafka/plugins,/opt/connectors,
#plugin.path=
tianzi 11:12 kafka_2.13-3.1.0 % 
```

Reply:
Please note that settings that are needed like `bootstrap.servers`, `key.converter` etc are kept as-is. We can add comments to explain what these settings are (and we already have them documented both on README and in the code) but please lets not comment them out. These properties need to be set in order to run the connector.

Reply:
Ah, good point. Would you mind uncomment them in a follow-up PR?

Reply:
Sure.